README.md

```diff
@@ -62,6 +62,7 @@ This constructor is used to transform an [API request](http://kuzzle.io/api-refe
 | `connectionId` | `string` | Passed to [RequestContext](#modelsrequestcontext) constructor |
 | `error` | `KuzzleError` or `Error` | Invokes [setError](#seterrorerror) at initialization |
 | `protocol` | `string` | Passed to [RequestContext](#modelsrequestcontext) constructor |
+| `headers` | `object` | Passed to [RequestContext](#modelsrequestcontext) constructor |
```

Comment:
@scottinet I just notice that with actual code, we cannot put in `options` arguments a parameter that will be passed to `RequestInput` constructor.

We have two choices :

* keep actual `RequestInput` constructor signature:
```javascript
// request.js
class Request {
  constructor(data, options) {
    this[_input] = new RequestInput(data);
    this[_context] = new RequestContext(options);
  }
}

// Construct a new request:
request = new Request(
  {controller: 'foo', action: 'bar', some: 'args', headers: {x-foo: 'baz'}}, 
  {protocol: 'xxx'}
);
```

* Add `options`, or at least `options.headers` to `RequestInput` constructor:

```javascript
// request.js
class Request {
  constructor(data, options) {
    this[_input] = new RequestInput(data, options.headers);
    this[_context] = new RequestContext(options);
  }
}

// requestInput.js
class RequestInput {
  constructor(data, headers {
// (...)
    this.body = data.body;
    this.headers = headers;
// (...)
  }
}

// Construct a new request:
request = new Request(
  {controller: 'foo', action: 'bar', some: 'args'}, 
  {protocol: 'xxx', headers: {x-foo: 'baz'}}
);
```

I'm divided between these 2 options...
What do you suggest ?